src/OpenLoco/src/Map/TileManager.cpp

```diff
@@ -464,16 +464,52 @@ namespace OpenLoco::World::TileManager
         return height;
     }
 
-    SmallZ getSurfaceCornerHeight(const SurfaceElement& surface)
+    static uint8_t getCornerDownMask(const uint8_t cornerUp)
+    {
+        switch (cornerUp)
+        {
+            case SurfaceSlope::CornerUp::north:
+                return SurfaceSlope::CornerDown::south;
+
+            case SurfaceSlope::CornerUp::south:
+                return SurfaceSlope::CornerDown::north;
+
+            case SurfaceSlope::CornerUp::west:
+                return SurfaceSlope::CornerDown::east;
+
+            case SurfaceSlope::CornerUp::east:
+                return SurfaceSlope::CornerDown::west;
+        }
+        return SurfaceSlope::flat;
+    }
+
+    // TODO: rename this function
+    SmallZ getSurfaceCornerHeight(const SurfaceElement& surface, const uint8_t cornerUp)
```

Comment:
`cornerUp` should be `cornerMask` to match the function declaration?

Reply:
Probably, yes, and I still want to rename the function so it isn't overloaded. Any suggestions?